org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/config/ZclClusterConfigHandler.java

```diff
@@ -32,16 +33,25 @@
      * Creates the list of {@link ConfigDescriptionParameter}. This method shall check the available attributes on the
      * remote device and create configuration parameters for each supported attribute that needs to be configurable.
      *
+     * @param cluster the {@link ZclCluster} to get the configuration
+     * @return true if this cluster has configuration descriptions
+     */
+    boolean initialize(ZclCluster cluster);
+
+    /**
+     * Gets the configuration description that was generated dynamically
+     *
      * @return the list of {@link ConfigDescriptionParameter}
      */
-    public List<ConfigDescriptionParameter> getConfiguration();
+    List<ConfigDescriptionParameter> getConfiguration();
```

Comment:
Yes, there might be room to improve this. I guess the question is if it is needed.

Reply:
I believe it might be beneficial in the case we are required to implement more config handler in the future. Since we currently only have two config handler, you are right and we should leave it as it is.